index.html

```diff
@@ -5,8 +5,35 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Ruda' rel='stylesheet'>
+    <link href="styles/index.css" rel="stylesheet">
+    
 </head>
 <body>
-    
+    <h1 id='display-city'>Seattle</h1>
+    <div id=pretty-picture>
```

Comment:
Minor note: you might consider making the div for each of the sections a `section` element